var/spack/repos/builtin/packages/quicksilver/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Quicksilver(MakefilePackage):
+    """Quicksilver is a proxy application that represents some elements of
+    the Mercury workload by solving a simpli?ed dynamic monte carlo
+    particle transport problem.  Quicksilver attempts to replicate the
+    memory access patterns, communication patterns, and the branching or
+    divergence of Mercury for problems using multigroup cross sections."""
```

Comment:
Can you separate the docstring into a short first sentence and a longer description in another paragraph?